### PR TITLE
deprecated kubectl run command flag replicas for zh

### PR DESCRIPTION
**Pull request by Cweiping**
[kubectl run ](https://github.com/kubernetes/kubernetes/issues/89651) become create pod ，so upgrade document of Chinese

Fix Issue：https://github.com/kubernetes/kubernetes/issues/91064

### CODE REVIEW

**Comment by tengqm** [comment]
Is this for 1.18 or 1.19?

**Comment by Cweiping** [comment]
> Is this for 1.18 or 1.19?
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"2cc6e0a70a", GitTreeState:"clean", BuildDate:"2020-03-26T06:42:04Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"2cc6e0a70a", GitTreeState:"clean", BuildDate:"2020-04-13T02:28:27Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
```
```
kubectl run nginx --image nginx --replicas 2
Flag --replicas has been deprecated, has no effect and will be removed in the future.
pod/nginx created
```

**Comment by sftim** [comment]
/hold

This should wait on PR #21536 landing, then localize those changes

File: content/zh/docs/reference/kubectl/docker-cli-to-kubectl.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ kubectl run --image=nginx nginx-app --port=80 --env="DOMAIN=cluster"
 -->
 ```shell
 # 启动运行 nginx 的 Pod
-kubectl run --image=nginx nginx-app --port=80 --env="DOMAIN=cluster"
+kubectl create deployment --image=nginx nginx-app --port=80 --env="DOMAIN=cluster"
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/docker-cli-to-kubectl.md:60]: do you mean we can specify `--port` in `kubectl` 1.18?

Comment (Cweiping) [reply content/zh/docs/reference/kubectl/docker-cli-to-kubectl.md]: Sorry, I didn't remove /hold after submitting the changes. Thank you for your timely review. The PR update has been completed.reference this [PR](https://github.com/kubernetes/website/pull/21536).

**Comment by tengqm** [comment]
Thanks for the fix.
/lgtm
/approve

**Pull request merged**